src/plugin/index.js

```diff
@@ -5,17 +5,19 @@ const importCwd = require("import-cwd"); // used to get the users project detail
 
 const pkg = importCwd("./package.json");
 
-function Jarvis(options) {
-
+function Jarvis(opts) {
```

Comment:
sorry I am bit annoying about this, in Node 6 and above we can do this:

`function Jarvis(options = {})` do you think we should do that?